apps/editor/src/markdown/plugins/syntaxHighlight.ts

```diff
@@ -130,15 +160,16 @@ function cacheIndexToRemoveBackground(doc: ProsemirrorNode, start: MdPos, end: M
 
   for (let i = start[0] - 1; i < end[0]; i += 1) {
     const node = doc.child(i);
-    const { codeStart, codeEnd } = node.attrs as CodeBlockPos;
+    // eslint-disable-next-line prefer-const
+    let { codeStart, codeEnd } = node.attrs as CodeBlockPos;
```

Comment:
이 경우는 컨벤션이 논의된 것이 있나요? codeStart는 변하지 않는다는 명확성을 위해서는 분리해서 선언하는 것도 어떨까 의견드립니다.

```suggestion
    const { codeStart } = node.attrs as CodeBlockPos;
    let { codeEnd } = node.attrs as CodeBlockPos;
```

Reply:
비효율적이어서 저는 그냥 합쳐서 사용하는데요. 좀 애매하긴 한 거 같아요.. 이 변수가 두 개뿐이라 부분은 분리할게요~

Reply:
넵 알겠습니다.

Reply:
이제와서 궁금한거지만, 그렇다면 해체 할당을 하지 않고 속성에 직접 접근하여 각각 변수 할당을 하는것은 컨벤션에 맞진 않는거죠?

```js
const codeStart = node.attrs.codeStart;
let codeEnd = node.attrs.codeEnd;
```

Reply:
아녀 이렇게 사용해도 됩니다. 다만 중복되는 코드가 많아지니까 해체 할당을 주로 쓰는데 위 같은 애매한 상황들이 오더라구요